test/visualTest.ts

```diff
@@ -1205,23 +1205,23 @@ describe("ChicletSlicer", () => {
 
                 Array.from(visualBuilder.slicerItemContainers)
                     .forEach((element: HTMLElement) => {
-                        expect(convertAnySizeToPixel(element.style.borderRadius)).toBeGreaterThan(0);
+                        expect(convertAnySizeToPixel(element.style.borderRadius, 0)).toBeGreaterThan(0);
```

Comment:
explain this change please

Reply:
ConvertAnySizeToPixel function expects two arguments, before change it received only 1 argument. Second argument refers to count of digits after decimal point. As we don't have any information about decimal in a test, and test worked without rounding, so that is why 0 should satisfy here.